drink/src/lib.rs

```diff
@@ -3,37 +3,27 @@
 
 #![warn(missing_docs)]
 
-mod bundle;
 pub mod errors;
-mod mock;
 pub mod runtime;
 pub mod sandbox;
 pub use sandbox::*;
 #[cfg(feature = "session")]
 pub mod session;
 
-use std::sync::{Arc, Mutex};
-
-pub use bundle::ContractBundle;
+#[cfg(feature = "session")]
 pub use drink_test_macro::{contract_bundle_provider, test};
```

Comment:
am I correct, that it is necessary to put only `contract_bundle_provider` behind feature gate? `test` should be fine

Reply:
hmm, yeah, that sounds more reasonable... especially that I don't like hiding macros behind `session` feature

so I suggest introducing second default feature `macros` and feature-gate reexport here (and the dependency as well)

Reply:
Done. I am testing this with ink! fyi, to make sure we don't forget stuff.
Quick question we should also build with just the std feature (the one we will use in ink!)
What's the best way to check that the code does not generate warning

just a new step in the rust-checks.yml?

Reply:
what warning do you have in mind? and yes, I guess adding some check to CI is the best idea